e2e/scripts/smoke_test.sh

```diff
@@ -61,6 +56,34 @@ rook_infra::run_test() {
     rc=$?; if [[ $rc != 0 ]]; then set -e; fi
 }
 
+rook_infra::try_copy_docker_image_to_rook_infra() {
```

Comment:
why do we need to load images explicitly? Aren't they already loaded from the build phase?